CMakeLists.txt

```diff
@@ -167,6 +167,18 @@ else()
     MESSAGE(FATAL_ERROR "Unknown build type '${CMAKE_BUILD_TYPE}'; valid types are 'Release' or 'Debug'")
 endif()
 
+# Sync with ShadowBuildCommon::cc_flags
+# TODO: move remaining C targets to Rust (even if just a Rust build script that
+# builds C files).
+add_compile_options(
+  # Always include debug symbols and frame pointers for ease of debugging
+  -ggdb
+  -fno-omit-frame-pointer
+  # Enable some particularly useful warnings
+  -Wreturn-type
+  -Wswitch
+)
```

Comment:
do these get passed on to the rust compiler somehow? does cmake put these in `CFLAGS` for example?

Reply:
Yup; added a comment to make this clearer.